test/unit/math/rev/fun/eigenvectors_sym_test.cpp

```diff
@@ -1,3 +1,4 @@
+#include <stan/math/rev.hpp>
```

Comment:
Why the changes to all these tests?

Reply:
Oof, these got changed when I was trying to fix the header include orders. I can remove them but tbh this does actually fix these so if you don't mind keeping them here I wouldn't mind leaving them

Reply:
I dont mind.